{
    "data_version": "4.0",
    "data_type": "CVE",
    "data_format": "MITRE",
    "CVE_data_meta": {
        "ID": "CVE-2023-23476",
        "ASSIGNER": "psirt@us.ibm.com",
        "STATE": "PUBLIC"
    },
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "IBM Robotic Process Automation 21.0.0 through 21.0.7.latest is vulnerable to unauthorized access to data due to insufficient authorization validation on some API routes.  IBM X-Force ID:  245425."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "CWE-200 Exposure of Sensitive Information to an Unauthorized Actor",
                        "cweId": "CWE-200"
                    }
                ]
            }
        ]
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "vendor_name": "IBM",
                    "product": {
                        "product_data": [
                            {
                                "product_name": "Robotic Process Automation",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_affected": "<=",
                                            "version_name": "21.0.0",
                                            "version_value": "21.0.7.latest"
                                        }
                                    ]
                                }
                            },
                            {
                                "product_name": "Robotic Process Automation for Cloud Pak",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_affected": "<=",
                                            "version_name": "21.0.0",
                                            "version_value": "21.0.7.latest"
                                        }
                                    ]
                                }
                            }
                        ]
                    }
                }
            ]
        }
    },
    "references": {
        "reference_data": [
            {
                "url": "https://www.ibm.com/support/pages/node/7017490",
                "refsource": "MISC",
                "name": "https://www.ibm.com/support/pages/node/7017490"
            },
            {
                "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/245425",
                "refsource": "MISC",
                "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/245425"
            }
        ]
    },
    "generator": {
        "engine": "Vulnogram 0.1.0-dev"
    },
    "source": {
        "discovery": "UNKNOWN"
    },
    "impact": {
        "cvss": [
            {
                "attackComplexity": "HIGH",
                "attackVector": "NETWORK",
                "availabilityImpact": "NONE",
                "baseScore": 3.1,
                "baseSeverity": "LOW",
                "confidentialityImpact": "LOW",
                "integrityImpact": "NONE",
                "privilegesRequired": "LOW",
                "scope": "UNCHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:L/I:N/A:N",
                "version": "3.1"
            }
        ]
    }
}